packages/ui/Changeable/Changeable.js

```diff
@@ -67,6 +67,7 @@ const defaultConfig = {
 const Changeable = hoc(defaultConfig, (config, Wrapped) => {
 	const {prop, change} = config;
 	const defaultPropKey = 'default' + cap(prop);
+	const getValue = props => props[prop] != null ? props[prop] : props[defaultPropKey];
```

Comment:
There's a slight change in functionality.  When controlled, it will go back to the default value if the prop is set to `null` or `undefined`.  Before, it would just set the value to `null` or `undefined`.  Is this change OK?

Reply:
I guess this could only happen if someone added the `[defaultPropKey]` value after initialization or specified both at startup.  Alternative is to move the `props[defaultPropKey]` check into `constructor()` and just return `props[prop]`.